index.js

```diff
@@ -1,2 +1,6 @@
 'use strict';
-module.exports = require(process.type === 'renderer' ? './source/renderer' : './source/main');
+
+module.exports.ipcRenderer =
+	process.type === 'renderer' ? require('./source/renderer') : undefined;
```

Comment:
I think this would look more readable:

```js
if (process.type === 'renderer') {
	module.exports.ipcRenderer = require('./source/renderer');
} else {
	module.exports.ipcMain = require('./source/main');
}
```